shesha-reactjs/src/designer-components/textArea/interfaces.ts

```diff
@@ -19,4 +19,5 @@ export interface ITextAreaComponentProps extends IConfigurableFormComponent {
   height?: string;
   width?: string;
   backgroundColor?: string;
-}
\ No newline at end of file
+  enForceSpellCheck?: boolean;
```

Comment:
Hi @MaseruKganyago. Please rename property to `spellCheck`  and update component settings as well